README.md

```diff
@@ -1,11 +1,17 @@
 # Castlevania: Symphony of the Night disassembly
 
-Recompilable code that creates 1:1 binaries for the commercial videogame Castlevania: Symphony of the Night for the PlayStation 1. This repository aims to create a full decompilation in C.
+the goal of this project is to create a full decompilation in C, that compiles the same binaries as the commercial video game Castlevania: Symphony of the Night for PS1.
```

Comment:
🔕  I think we can omit "the goal of this project is". Let's just go with "Creates a full decompilation in C"